_events/2022/2022-03-dei-diverse-leung.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Normalizing inclusion by embracing difference
+subtitle: DiveRSE Speaker Series
+expires: 2022-03-23
+event_date: "March 22, 2022"
+layout: event
+duration: 60
+repeated: false
+category: dei 
+time:
+    - - start: 2022-03-11T16:00:00Z
+        end: 2022-03-11T17:30:00Z
+---
+
+## DiveRSE Speaker Series Info
+
+DiveRSE (Diverse RSE) is a series of talks and related activities including discussion and panel sessions designed to support and raise awareness of EDI challenges and successes within the research software community.
```

Comment:
```suggestion
DiveRSE (Diverse RSE) is a series of talks and related activities including discussion and panel sessions designed to support and raise awareness of DEI challenges and successes within the research software community.
```

Reply:
Actually I think this is correct - either write it out here and add (EDI) or just write it out and define the acronym later. But we can't use the acronym without defining it first!